Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/triangulate_faces.h

```diff
@@ -45,11 +45,32 @@ namespace CGAL {
 
 namespace Polygon_mesh_processing {
 
+namespace Triangulate_faces
+{
+/** \ingroup PMP_meshing_grp
+*   Default new face visitor model of `PMPTriangulateFaceVisitor`.
+*   All its functions have an empty body. This class can be used as a
+*   base class if only some of the functions of the concept require to be
+*   overriden.
+*/
+template<class TriangleMesh>
+struct Default_visitor {
```

Comment:
Without namespace this name has to change.   So maybe the PMP package has already namespaces with the same class name.

Reply:
Exactly. There is one in namespace `Corefinement`. I followed the same scheme.